dvc/remote/base.py

```diff
@@ -61,11 +61,19 @@ def __init__(self, remote, cmd, ret, err):
 
 
 class RemoteActionNotImplemented(DvcException):
-    def __init__(self, action, scheme):
+    def __init__(self, action, scheme, hint=None):
         m = "{} is not supported by {} remote".format(action, scheme)
+
+        if hint:
+            m += ". {}".format(hint)
+
         super(RemoteActionNotImplemented, self).__init__(m)
 
 
+class RemoteConfigError(DvcException):
```

Comment:
I would either simply use ConfigError or at least would inherit from it.

Reply:
:eyes: forgot about `ConfigError`, I'll use it instead